index.html

```diff
@@ -0,0 +1,334 @@
+            >
+          </div>
+          <div class="hero__info-box">
+            <span class="hero__info-title">Challenge yourself</span>
+            <p class="hero__info-text">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas
+              fringil. lacus eget consequat tincidunt, lectus ex.
+            </p>
+            <a class="hero__info-link" href="#"
+              ><span
+                ><img
+                  class="hero__info-link-image"
+                  src="images/hero-arrow.png"
+                  alt=""
+                />Explore more</span
+              ></a
+            >
+          </div>
+        </section>
+      </section>
+      <section class="training">
+        <h2 class="training__title">latest training</h2>
+        <section class="training__teaser">
+          <a class="training__teaser-link" href="#">
+            <img
+              class="training__image"
+              src="images/training-running.png"
+              alt=""
+            />
+            <span class="training__image-title">running</span>
+            <span class="hide-mobile training__teaser-text"
+              >Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas
+              fringilla, lacus eget consequat
+            </span>
+            <span class="button maxwidth-100 hide-mobile">See more</span>
+          </a>
+        </section>
+        <section class="training__teaser">
+          <a class="training__teaser-link" href="#">
+            <img
+              class="training__image"
+              src="images/training-lifting.png"
+              alt=""
+            />
+            <span class="training__image-title">Lifting</span>
+            <span class="training__teaser-text hide-mobile"
+              >Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas
+              fringilla, lacus eget consequat
+            </span>
+            <span class="button maxwidth-100 hide-mobile">See more</span>
+          </a>
+        </section>
+        <section class="training__teaser">
+          <a class="training__teaser-link" href="#">
+            <img
+              class="training__image"
+              src="images/training-yoga.png"
+              alt=""
+            />
+            <span class="training__image-title">yoga</span>
+            <span class="training__teaser-text hide-mobile"
+              >Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas
+              fringilla, lacus eget consequat
+            </span>
+            <span class="button maxwidth-100 hide-mobile">See more</span>
+          </a>
+        </section>
+        <a class="button hide-desktop" href="#">See more</a>
+        <a class="training__link-category hide-mobile" href="#">See more</a>
+      </section>
+      <section class="team">
+        <h2 class="team__heading">our team</h2>
+        <ul class="team__list" role="list">
+          <li class="team__list-item">
```

Comment:
Better name this `team__teaser` or similar. It does not matter that it is a list-item. What if I use this styles on another element?